packages/api/src/routes/medical/document.ts

```diff
@@ -105,20 +126,92 @@ router.post(
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getCxIdOrFail(req);
     const patientId = getFromQueryOrFail("patientId", req);
-    const facilityId = getFrom("query").optional("facilityId", req);
+    const facilityId = getFromQueryOrFail("facilityId", req);
     const override = stringToBoolean(getFrom("query").optional("override", req));
+    const docRequestMetadata = docRequestMetadataDataSchemaOptional.parse(req.body);
+
+    const requestCreate: DocRequestCreateCmd = {
+      cxId,
+      facilityId,
+      patientId,
+      metadata: { data: docRequestMetadata.data || {} },
+    };
+
+    // TODO pass REQUEST into queru.
+    // TODO create DocQueryProgress Stuff
+
+    const createdDocRequest: DocRequest = await createDocRequest(requestCreate);
+    console.log("Request", JSON.stringify(createdDocRequest));
 
     const docQueryProgress = await queryDocumentsAcrossHIEs({
       cxId,
       patientId,
       facilityId,
+      docRequest: createdDocRequest,
       override,
     });
 
     return res.status(OK).json(docQueryProgress);
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * PUT /document/request/:id
+ *
+ * Updates the request according to the request ID
+ * Note: this is not a PATCH, so requests must include all patient data in the payload.
+ *
+ * @param req.query.requestId The facility providing NPI for the patient update
+ * @return The patient to be updated
+ */
+router.put(
+  "/:id",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getCxIdOrFail(req);
+    const requestId = getFromQueryOrFail("requestId", req);
+    const docRequestMetadata = docRequestMetadataDataSchemaRequired.parse(req.body);
```

Comment:
Metadata mandatory because thats the only thing to update.
Unsure if this endpoint is something we actually want to support.